data/assets/examples/scale/timelinescale/no-interpolate.asset

```diff
@@ -0,0 +1,52 @@
+-- No Interpolation
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The scale of
+-- the coordinate axes are determined by a timeline of individual scales that are used
+-- without interpolating between the timeline entries. These scales are keyframes that are
+-- used to change between different sizes. This example transitions between three sizes.
+-- In this example, the interpolation between entries is disabled, which will cause the
+-- coordinate axes to change their size abruptly when the scale changes. If the
+-- interpolation were enabled, the orientation of the coordinate axes would transition
+-- seamlessly instead at the provided times. This example will only work if the in-game
+-- time is set to January 1st, 2000.
+
+local Node = {
+  Identifier = "TimelineScale_Example_NoInterpolation",
+  Transform = {
+    Scale = {
+      Type = "TimelineScale",
+      Keyframes = {
+        -- The first timeline entry
+        ["2000 JAN 01 00:00:00"] = {
+          Type = "StaticScale",
+          Scale = 10.0
+        },
+        -- The second timeline entry
+        ["2000 JAN 01 12:00:00"] = {
+          Type = "StaticScale",
+          Scale = 0.0
```

Comment:
```suggestion
          Scale = 10.0
```
I think it will be very strange for the user if the example disappears due to the 0 scale, so I suggest an alternative timeline. Since this example does not interpolate, it is going to stay "disappeared" for quite a long time in contrast to the example that interpolates.